more/free-programming-interactive-tutorials-en.md

```diff
@@ -219,17 +220,22 @@
 
 #### jQuery
 
-* [Front End Libraries: jQuery](https://www.freecodecamp.org/learn/front-end-libraries/jquery)
+* [Front End Development Libraries Certification: jQuery](https://www.freecodecamp.org/learn/front-end-libraries/#jquery) - freeCodeCamp
```

Comment:
Use each course url instead of its track from belongs to

```suggestion
* [Front End Development Libraries Certification: jQuery](https://www.freecodecamp.org/learn/front-end-libraries/jquery) - freeCodeCamp
```